src/main.cpp

```diff
@@ -90,6 +90,7 @@ static std::string parse_commandline(int argc, char *argv[]) {
 #endif
 #ifdef USE_TUNER
         ("puct", po::value<float>())
+        ("fpu_reduction", po::value<float>())
```

Comment:
Consider making dynamic eval enable/disable at the same time?

Reply:
Good idea, I'll put it in tomorrow since I can't code at the moment.